docs/src/algorithms/discretization.md

```diff
@@ -3,5 +3,68 @@
+
+mesh = discretize(poly, FIST()) # hide
+
+plot(plot(poly), plot(mesh)) # hide
+```
+
+```@example FIST
+poly = PolyArea(Point2f[(0.18142937, 0.54681134), (0.38282228, 0.107781954), (0.43220532, 0.013640274), (0.48068276, 0.019459315), (0.48322055, 0.11583236), (0.46696007, 0.2230227), (0.48184678, 0.2656454), (0.45998818, 0.2784367), (0.4168235, 0.2190962), (0.4124987, 0.21208182), (0.39593673, 0.2520411), (0.44333926, 0.28375763), (0.4978224, 0.3981428), (0.7703431, 0.20181546), (0.7612364, 0.33008572), (0.9856581, 0.2215304), (0.99374324, 0.3353423), (0.87789994, 0.32551613), (0.5614043, 0.540334), (0.9494598, 0.39622766), (0.9688778, 0.38663587), (0.59554976, 0.655444), (0.59496254, 0.58492756), (0.27641845, 0.656314), (0.3242084, 0.6072907), (0.42408508, 0.49353212), (0.20984341, 0.59003067), (0.2799388, 0.52516246), (0.38555774, 0.32233855), (0.36943135, 0.30108362), (0.18142937, 0.54681134)], []) # hide
+
+mesh = discretize(poly, FIST()) # hide
+
+plot(plot(poly), plot(mesh)) # hide
+```
+
+```@example FIST
+poly = PolyArea(Point2f[(0.18142937, 0.54681134), (0.38282228, 0.107781954), (0.43220532, 0.013640274), (0.48068276, 0.019459315), (0.48322055, 0.11583236), (0.46696007, 0.2230227), (0.48184678, 0.2656454), (0.45998818, 0.2784367), (0.4168235, 0.2190962), (0.4124987, 0.21208182), (0.39593673, 0.2520411), (0.44333926, 0.28375763), (0.4978224, 0.3981428), (0.7703431, 0.20181546), (0.7612364, 0.33008572), (0.9856581, 0.2215304), (0.99374324, 0.3353423), (0.9688778, 0.38663587), (0.59554976, 0.655444), (0.59496254, 0.58492756), (0.27641845, 0.656314), (0.3242084, 0.6072907), (0.42408508, 0.49353212), (0.20984341, 0.59003067), (0.18142937, 0.54681134)], []) # hide
+
+mesh = discretize(poly, FIST()) # hide
+
+plot(plot(poly), plot(mesh)) # hide
+```
+
+```@example FIST
+poly = PolyArea(Point2f[(0.20974362, 0.11143557), (0.24619843, 0.019633872), (0.2525517, 0.10491638), (0.2605699, 0.03590731), (0.2794662, 0.009585844), (0.54061747, 0.2523168), (0.5210726, 0.27176696), (0.55140936, 0.29682124), (0.5437957, 0.24153952), (0.6285009, 0.21651964), (0.6449456, 0.2147006), (0.59213245, 0.26681182), (0.8229673, 0.74253976), (0.65908766, 0.56879747), (0.5620604, 0.53223926), (0.52954125, 0.5394655), (0.23878936, 0.1011955), (0.22548631, 0.09457563), (0.2759589, 0.71809477), (0.5383939, 0.85454345), (0.6496192, 0.9248424), (0.6248543, 0.8470848), (0.749698, 0.8314047), (0.75198966, 0.9489501), (0.7027816, 0.92791766), (0.7210894, 0.8686051), (0.65642047, 0.9434703), (0.6423689, 0.9363239), (0.549272, 0.9676014), (0.23183312, 0.71278405), (0.20974362, 0.11143557)], []) # hide
+
+mesh = discretize(poly, FIST()) # hide
+
+plot(plot(poly), plot(mesh)) # hide
+```
```

Comment:
I think we only need one example, or perhaps two with different features. One example where polygonal area doesn't have holes (inner rings) and another example with holes.

Reply:
Thank you, I will work on this immediately.